projects/storefrontlib/src/cms-components/misc/qualtrics/config/default-qualtrics-config.ts

```diff
@@ -1,5 +1,7 @@
 import { QualtricsConfig } from './qualtrics-config';
-
+/**
+ * @deprecated since 3.1 - moved to feature-lib
```

Comment:
Maybe we should briefly describe what can customers do to _not_ use the deprecated code.
This applies to all other deprecation warnings.

Some suggestions:
- point to the docs where we explain how to migrate
- Write some schematics that will perform the automatic migration (not sure about the work required to achieve this. we can discuss it)
- Briefly explain it here (repetitive)

Reply:
added point to docs where we explain how to migrate

docs will contain the how to